### PR TITLE
OCPBUGS-10437: Fix CNI conf

### PR DESCRIPTION
**Pull request by aravindhp**
https://github.com/openshift/windows-machine-config-operator/commit/62dc2c3e8db93de37a3b557c52de7bb18d3bfd85 introduced creating a CNI config with PascalCase moving away from the camelCase that was used previously generated. Generated is the key word here as the old template used PascalCase but the generated config was camelCase. This commit fixes the issue by adhering to the CNI config [definition](https://www.cni.dev/plugins/current/main/win-overlay/) for the win-overlay plugin.

### CODE REVIEW

File: pkg/nodeconfig/payload/payload.go [hunk 1]
```diff
@@ -80,49 +80,49 @@ Import-Module -DisableNameChecking HNS_MODULE_PATH
 
 $cni_template=@'
 {
-    "CniVersion":"0.2.0",
```
[the remark this answers is not in the document]
Comment (aravindhp) [reply pkg/nodeconfig/payload/payload.go:83]: Given how close we are to a release and this is a blocker bug, I am not comfortable making that level of a change. I suggest we do this as a follow up PR.

[the remark this answers is not in the document]
Comment (sebsoto) [reply pkg/nodeconfig/payload/payload.go:83]: Thats fine with me

**Comment by mansikulkarni96** [comment]
@aravindhp [unit tests] (https://github.com/openshift/windows-machine-config-operator/blob/c85720293bf3e85c38472e9d1b0ac500d800161f/pkg/nodeconfig/payload/payload_test.go#L15)will also need to be adjusted to match the new casing.

**Comment by aravindhp** [comment]
> @aravindhp [unit tests](https://github.com/openshift/windows-machine-config-operator/blob/c85720293bf3e85c38472e9d1b0ac500d800161f/pkg/nodeconfig/payload/payload_test.go#L15) will also need to be adjusted to match the new casing.

@mansikulkarni96 Thanks for catching. Fixed.

**Review by saifshaikh48 — COMMENTED** [review]
/lgtm

**Pull request merged**